### ISSUE TITLE
Crash in setting dialog when built as x64 

### ISSUE THREAD

**Issue by edwpang** [Predelnik/DSpellCheck#76]
```
   ETDTProc **_enableDlgTheme_** = (ETDTProc)::SendMessage(
           _hParent, NPPM_GETENABLETHEMETEXTUREFUNC, 0, 0);
  if (enableDlgTheme) enableDlgTheme(_hSelf, ETDT_ENABLETAB);
```

The enableDlgTheme is being truncated to 32bit because of the following code DllMain.cpp:
LRESULT CALLBACK SubWndProcNotepad(HWND hWnd, UINT Message, WPARAM wParam, LPARAM lParam)
{
  Int ret; 
....
Changed the return to LRESULT fixes the problem:

  LRESULT ret;

**Comment by chcg** [comment]
@Predelnik Probably fixed with release v1.3.0.

**Comment by Predelnik** [comment]
Yeah I think looks like it's fixed, so I'm closing. Feel free to reopen if any x64 specific issue still remain.
